rslib/src/scheduler/fsrs/params.rs

```diff
@@ -55,6 +55,7 @@ impl Collection {
     /// Note this does not return an error if there are less than 400 items -
     /// the caller should instead check the fsrs_items count in the return
     /// value.
+    #[allow(clippy::too_many_arguments)]
```

Comment:
I'm guilty of doing this too, but let's do it properly and split the args into a struct instead.